.changeset/long-files-raise.md

```diff
@@ -0,0 +1,5 @@
+---
+'@harnessio/oats-plugin-react-query-harness': minor
```

Comment:
this looks like a breaking change.. should be a major version bump I think

Reply:
this ID was always missing in scopedGroups and we never use this ID manually anywhere else , used internally by useQuery as query key

Reply:
this shouldn't be a breaking change